### DISCUSSION TITLE
How to change Development Command in production??

### DISCUSSION THREAD

**Discussion by yicheng-blockin** [vercel/vercel#8745] [Help] [answered]
I want use "vite preivew" instead of "vite"

**Comment by amyegan** [comment, marked as answer]
Hi @yicheng-blockin! `vite preview` is intended for use in local environments to preview builds. It will not work in a production deployment environment because of differences in architecture and permissions. If you can share more about why you want to use `vite preview` in production, I can help you come up with a different solution.

More info:
- https://stackoverflow.com/a/71703991
- [Testing the App Locally](https://vitejs.dev/guide/static-deploy.html#testing-the-app-locally)
